app/services/deploy_service.rb

```diff
@@ -48,6 +48,7 @@ def create_github_pull_request
           return
         end
       end
+      can_release_now?(deploy_strategy.arguments['blocked_time_buckets'], merge_at)
```

Comment:
This comes a few lines _after_ the release PR is merged. It should be checked _before_ merging and ideally before creating new release PRs.

Also, shouldn't `Time.now` be used rather than `merge_at` (which is some past time)?

Reply:
Hi @joeyAghion , I used merge_at actually because I assumed that we where more interested in the time this PR is planned to be merged, I will change it to use now instead.
I will also put the check before the merge, I think I mixed the version somehow, probably doing some rollback on the tests I accidentally put it back on the wrong place.

Reply:
Just pushed the changes :)